portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,11 +38,28 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String currComment = request.getParameter("comment");
-    writeComments(currComment);
-    String json = convertToJson(readComments());
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
+    String translatedText = translateComment(request);
+    if(translatedText != "") {
+        writeComments(translatedText);
```

Comment:
Just a suggestion and something for you to think about: You are storing the translated text to datastore, but not storing the language that it was translated to. It might be better to write the original text and translate on doGet.
Is the expected behavior that the user types comments in english and want to display in a different language?

Reply:
yes, the expected behavior is that the user is able to only write in english and then have their comment translated to other languages. This may change as I make changes to my portfolio